src/common/components/AddTodoModal/AddTodoModal.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { useState } from "react";
+import {
+  Dialog,
+  DialogTitle,
+  DialogContent,
+  DialogActions,
+  TextField,
+  Button,
+  FormControl,
+  InputLabel,
+  Select,
+  MenuItem,
+  CircularProgress,
+} from "@mui/material";
+// import { selectAllUsers, selectUsersLoading } from "../../app/store/selectors/UserSelectors";
+import { createTask } from "../../../app/store/reducers/ActionCreators";
+import { IUser } from "../../../features/users/types";
+import { useAppDispatch, useAppSelector } from "../../../app/hooks/redux";
+import { selectAllUsers, selectUsersLoading } from "../../../features/users/selectors";
+
+interface AddTodoModalProps {
+  open: boolean;
+  onClose: () => void;
+}
+
+const AddTodoModal: React.FC<AddTodoModalProps> = ({ open, onClose }) => {
+  const dispatch = useAppDispatch();
+  const users = useAppSelector(selectAllUsers);
+  const isLoading = useAppSelector(selectUsersLoading);
+
+  const [title, setTitle] = useState("");
+  const [userId, setUserId] = useState<number | "">("");
+
+  const handleSubmit = () => {
+    if (!title || userId === "") return;
+
+    dispatch(createTask({ title, userId, completed: false }));
+    setTitle("");
+    setUserId("");
+    onClose();
```

Comment:
remind me about it during the call
there is an important thing regarding it

Reply:
setTitle("");
setUserId("");

These methods shouldn't be called as **onClose()** already closes this modal and all local state will be automatically reseted to default